CITATION.cff

```diff
@@ -0,0 +1,42 @@
+abstract: "EICrecon is the EIC online/offline reconstruction software."
+authors: # in the order of https://zenodo.org/records/15880663/export/cff
+- given-names: The ePIC Collaboration
+- family-names: Wouter Deconinck
+- family-names: David Lawrence
+- family-names: Dmitry Romanov
+- family-names: Dmitry Kalinkin
+- family-names: Christopher Dilks
+- family-names: Thomas Britton
+- family-names: Simon Gardner
+- family-names: Tooba Ali
+- family-names: Nathan Brei
+- family-names: Xinxin (Cissie) Mei
+- family-names: Derek M Anderson
+- family-names: Barak Schmookler
+- family-names: Ryan Milton
+- family-names: Sebouh Paul
+- family-names: Alex Jentsch
+- family-names: John Lajoie
+- family-names: Sakib Rahman
+- family-names: Joseph (Joe) Osborn
+- family-names: Shyam Kumar
+- family-names: Tommy Chun Yuen Tsang
+- family-names: Zhongling Ji
+- family-names: Matt Posik
+- family-names: Peter Steinberg
+- family-names: Minho Kim
+- family-names: Daniel Brandenburg
+- family-names: Dhevan Gangadharan
+cff-version: 1.2.0
+date-released: '<replace with the actual release date>'
```

Comment:
Omit these?

Reply:
I assume if we omit, Zenodo will fill this in?

Looking at https://github.com/scikit-hep/uproot5/blob/main/CITATION.cff
they put date-released for the original release date and version is not specified.

Each version on Zeonodo is already provided with an unique DOI, we don't need to make a CITATION.cff to address that.

Reply:
My concern is that people won't (only) go to zenodo for the citation, but copy it directly from GitHub, see https://github.com/eic/epic/ and the "Cite this repository" menu on the right sidebar. That's based purely on what's inside the cff file.

Reply:
So we should be removing the cff file from eic/epic and replacing it with a Zenodo link in the README?

Reply:
That's what we have here... and it isn't working. I think having the "Cite this repository" is a better solution than expecting people to know they can cite zenodo. They (especially now) think of zenodo as a place for internal documents.